### DISCUSSION TITLE
WslRegisterDistribution failed with error: 0x80370102

### DISCUSSION THREAD

**Discussion by azeezz0090** [Cyfrin/foundry-full-course-cu#2116] [Q&A] [answered]
Pls someone should help me out with this error:

```
Ubuntu is already installed.
Launching Ubuntu...
Installing, this may take a few minutes...
WslRegisterDistribution failed with error: 0x80370102
Please enable the Virtual Machine Platform Windows feature and ensure virtualization is enabled in the BIOS.
For information please visit https://aka.ms/enablevirtualization
Press any key to continue...
```
I have enabled the Virtual Macjine Platform Windows Features.

**Comment by EngrPips** [comment, marked as answer]
Hello @azeezz0090, Please visit this `https://answers.microsoft.com/en-us/windows/forum/all/wslregisterdistribution-failed-with-error/5fdd8aa6-175e-4a14-8fe6-07673133fd15` and see if it will help you resolve this issue.

**Reply by JoshuyPan** [reply]
you're a sniper man, always faster than me!

**Reply by EngrPips** [reply]
Not every time. I used to be slow sometimes, too.

**Reply by JoshuyPan** [reply]
you're amazing, keep it up buddy

**Reply by EngrPips** [reply]
I will. Thanks for the good words.

**Reply by azeezz0090** [reply]
@EngrPips Thank you Sir

**Reply by EngrPips** [reply]
You are welcome, Friend. Keep crushing it.
